packages/storybook/stories/INTERNAL/Survey/TagsSurvey.stories.tsx

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { mergeStyles } from '@fluentui/react';
+import {
+  _TagsSurvey as TagsSurveyComponent,
+  _AudioIssue,
+  _OverallIssue,
+  _ScreenshareIssue,
+  _SurveyTag,
+  _VideoIssue,
+  useTheme
+} from '@internal/react-components';
+import { Canvas, Description, Heading, Props, Title } from '@storybook/addon-docs';
+import { Meta } from '@storybook/react/types-6-0';
+import React, { useState } from 'react';
+import { COMPONENT_FOLDER_PREFIX } from '../../constants';
+import { hiddenControl } from '../../controlsUtils';
+import { TagsSurveyExample } from './snippets/TagsSurvey.snippet';
+
+const ExampleSurveyText = require('!!raw-loader!./snippets/TagsSurvey.snippet.tsx').default;
+
+const getDocs: () => JSX.Element = () => {
+  /* eslint-disable react/no-unescaped-entities */
+  return (
+    <>
+      <Title>Tags Survey</Title>
+      <Description>Component to render a modal containing a Tags Survey</Description>
+
+      <Heading>Usage</Heading>
+      <Description>Here is an example of how to use `TagsSurvey`</Description>
+      <Canvas mdxSource={ExampleSurveyText}>
+        <TagsSurveyExample />
+      </Canvas>
+
+      <Heading>Props</Heading>
+      <Props of={TagsSurveyComponent} />
+    </>
+  );
+};
+
+const TagsSurveyStory = (): JSX.Element => {
+  const [showSurvey, setShowSurvey] = useState(true);
+  const theme = useTheme();
+  const onDismiss = (): void => {
+    setShowSurvey(false);
+  };
+
+  const strings = {
+    TagsSurveyQuestion: 'What could have been better?',
+    TagsSurveyConfirmButtonLabel: 'Submit Survey',
+    TagsSurveyCancelButtonLabel: 'Cancel',
+    cancelButtonAriaLabel: 'Cancel'
+  };
+
+  const callIssuesToTag = {
+    overallRating: {
+      CallCannotJoin: 'I could not join call',
+      CallCannotInvite: 'I could not invite others into the call',
+      HadToRejoin: 'I had to rejoin the call',
+      CallEndedUnexpectedly: 'Call ended for me unexpectedly',
+      OtherIssues: 'I was having other issues with the call'
+    },
+    audioRating: {
+      NoLocalAudio: 'The other side could not hear any sound',
```

Comment:
are these going to be localized? or is this something that only exists for Contoso to add their own strings to?